tock/tock/templates/400.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h2>Bad Request</h2>
```

Comment:
Let's ping someone from content or writing lab and see if they have any suggestions on more user-friendly copy here.

Reply:
@amandacostello can be of assistance. Looping in.

Reply:
@amandacostello any update?

Reply:
Hi! Still getting my feet under me with GitHub. You're looking for copy related to just 400 error returns, or also 403, 404, and 500?

Reply:
In particular, 400, but if there was general guidance for the others we'd be happy to incorporate that as well.

Reply:
I've got some writing for each, though what usually triggers the 400 on the client side? Also, as a new GitHub person, how/where should I put the new content? Or just put it in a comment here?

Reply:
### Error: Bad Request (400)
We can't process your request due to something wrong on your end, like a message being too large or (other common example seen in Tock)

### Error: Access forbidden (403)
Your account doesn't have access. Make sure you're logged in, check your VPN connections, and that your account has the correct permissions enabled. 

### Error: File not found (404)
Looks like the file you're requesting isn't here anymore. Contact (name/email) or try (search)

### Error: Internal Server Error (500)
Something's gone wrong on our side. Contact (name/email) to let them know.